### PR TITLE
Add an option to create modals from JSX Elements

### PR DESCRIPTION
**Pull request by Schottkyc137**
This PR is a proposal on how one could add modals to the project.

Please review it carefully to make sure it adheres to your code style. I'm not quite sure how to write a test-case for the modals as well

### CODE REVIEW

**Comment by vacarsu** [comment]
Hey sorry it's taken me so long to get to this. Can you add this to the storybook?

**Review by vacarsu — CHANGES_REQUESTED** [review]

File: src/components/Modal/Modal.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ *         <ModalTitle>Hello, World!</ModalTitle>
+ *         <p>Hello from Modal</p>
+ *     </ModalBody>
+ * ).show()
+ * </code></pre>
+ * @param content The content to display
+ * @param options Modal options
+ */
+export function modal(content: JSX.Element, options?: UIkit.ModalOptions): UIkit.ModalElement {
+  const divElement = document.createElement('div');
```
[the remark this answers is not in the document]
Comment (Schottkyc137) [reply src/components/Modal/Modal.tsx:25]: I'm not quite sure what you mean by this. The `div` element I'm creating is the container for the modal. Can you show me a more elaborate example of what you mean?

**Comment by Schottkyc137** [comment]
> Hey sorry it's taken me so long to get to this. Can you add this to the storybook?

Added it to storybook. Nice to see that you are still maintaining this!

**Review by vacarsu — APPROVED** [review]

**Comment by vacarsu** [comment]
I pulled the code ran the storybook. I see what you're doing now after looking more thoroughly. Thanks for this!
This closes #34

**Pull request merged**